content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+Multiple meshes afford the following capabilities beyond that of a single mesh:
+
+- Organizational boundaries: lines of business
+- Service name or namespace collisions: multiple distinct uses of the `default`
+  namespace
+- Stronger isolation: isolating test workloads from production workloads
+
+In the most basic multiple mesh model, each mesh forms a strict
+boundary and there is no communication between meshes. However, you can enable
+inter-mesh communication with **mesh peering**. When
+peering, each mesh can export a set of services and identities, which all
+participating meshes can recognize.
+
+{{< image width="50%"
+    link="./multi-mesh.svg"
+    alt="Multiple service meshes"
+    title="Multi-mesh"
+    caption="Multiple service meshes"
+    >}}
+
+To avoid service naming collisions, you can give each mesh a globally unique
+**mesh ID**, to ensure that the fully qualified domain
+name (FQDN) for each service is distinct.
+
+When peering two meshes that do not share the same
+{{< gloss >}}trust domain{{< /gloss >}}, {{< gloss >}}identity{{< /gloss >}} and
+**trust bundles** must be federated between them. See the section
+on [Multiple Trust Domains](#trust-between-meshes) for an overview.
+
+## Control plane models
+
+The Istio [control plane](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-control-plane)
+configures the mesh or a subset of the mesh to manage the communication between
+workload instances. A single deployment of the control plane consists of several
+redundant processes that share a common view of the mesh and provide workload
+instances with identical configuration.
+
+Depending on your overall system architecture and cloud provider, the following
+Istio control plane models are possible:
+
+- [single control plane](#single-control-plane)
+- [multiple control planes](#multiple-control-planes)
+- [managed control planes](#managed-control-plane)
+
+### Single control plane
+
+In the simplest case, you can run your mesh using a single control plane
+deployment. The single control plane model is typically used for [single
+cluster](#single-cluster) meshes.
+
+If you run your mesh on a single cluster, a single control plane deployment
+suffices. Each workload instance in the cluster connects to a control plane
+instance to get its configuration.
+
+{{< image width="50%"
+    link="./single-cluster.svg"
+    alt="A service mesh with a single control plane"
+    title="Single control plane"
+    caption="A service mesh with a single control plane"
+    >}}
+
+### Multiple control planes
+
+You can deploy multiple control planes across the mesh; such a configuration is
+known as a **multi-control plane**.
+
+Multiple control planes afford improved performance and availability for meshes
+that span [multiple clusters](#multiple-clusters) across multiple zones and
+regions.
+
+Istio supports control plane deployments at various levels of availability:
+
+- Control plane per region: *lowest availability*
```

Comment:
So in this model, the control planes are shared between the clusters? There is a single control plane per region that is shared between the clusters in the region?

Reply:
Yes that's right, and the single control plane per region may run in multiple clusters for HA. So its per-region shared control planes.

Reply:
I see. So we can have a single control plane, and multiple control planes, which can be shared or dedicated. cc @frankbu . But should we, if so, split the multiple control planes into shared/dedicated subsections?

Reply:
@nmittler @smawson @rcaballeromx @vadimeisenbergibm Can we please stop using the term `single control plane`? I've had several long discussion with @costin in various design docs to come to a common understanding that single control plane makes no sense in production so we should not describe it as a realistic configuration.

I think the control plane section intro should just say something like:

You can have one or more control planes (usually more for HA).

Then the following sections describe these three kinds of control plane configurations:

1. dedicated (manage workloads in only one cluster)
2. shared (manage workloads in more than one cluster)
3. managed

We would explain how using multiple control planes offer HA and that they would be synchronized (replicated types) if they are in a single multi-cluster mesh. Control planes in a multi-mesh configuration, on the other hand, would have different types in each mesh's control plane(s).